nav2_behavior_tree/include/nav2_behavior_tree/bt_action_node.hpp

```diff
@@ -340,6 +357,12 @@ class BtActionNode : public BT::ActionNodeBase
           result_ = result;
         }
       };
+    send_goal_options.feedback_callback =
+      [this](typename rclcpp_action::ClientGoalHandle<ActionT>::SharedPtr,
+             const std::shared_ptr<const typename ActionT::Feedback> feedback) {
```

Comment:
`spin()` happens during `tick()` and specific function calls, so I don't think there's an issue with async access to `feedback_` to need a mutex. Do you agree?

Reply:
Yup, makes sense.